ada_2_fhir-r4/cio/2.0.0/beschikbaarstellen_medicatie_contra_indicaties/fhir_instance/mci-pat03-mci-zwa-01.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Flag xmlns="http://hl7.org/fhir">
+   <id value="mci-pat03-mci-zwa-01"/>
+   <meta>
+      <profile value="http://nictiz.nl/fhir/StructureDefinition/cio-MedicationContraIndication"/>
+   </meta>
+   <identifier>
```

Comment:
extension RelationAlert is missing, but not required, however occurs in ada instance)